pkg/backup/item_collector.go

```diff
@@ -532,28 +705,72 @@ func (r *itemCollector) listItemsForLabel(unstructuredItems []unstructured.Unstr
 }
 
 // backupNamespaces process namespace resource according to namespace filters.
-func (r *itemCollector) backupNamespaces(unstructuredList *unstructured.UnstructuredList,
-	ie *collections.IncludesExcludes, gr schema.GroupResource, preferredGVR schema.GroupVersionResource,
-	log logrus.FieldLogger) []*kubernetesResource {
-	var items []*kubernetesResource
-	for index, unstructured := range unstructuredList.Items {
-		if ie.ShouldInclude(unstructured.GetName()) {
-			log.Debugf("Backup namespace %s due to namespace filters setting.", unstructured.GetName())
+func (r *itemCollector) backupNamespaces(
```

Comment:
This should be renamed as it's still in the "collecting" phase. 

I find it a little confusing for its calling the nsTracker.init, b/c the tracker is not really respected in this func.

Reply:
Renamed to `collectNamespaces`.